File: 240115/시력 검사/eye-test.py
arr = input().split()

left = float(arr[0])
right = float(arr[1])

if left >= 1.0 and right >= 1.0:
    print("High")
elif left >= 0.5 and right >= 0.5:
    print("Middle")
else:
    print("Low")

File: 240409/시력 검사/eye-test.swift
var leftEye = Double(readLine()!)!
var rightEye = Double(readLine()!)!

if leftEye >= 1.0 && rightEye >= 1.0 {
    print("High")
} else if leftEye >= 0.5 && rightEye >= 0.5 {
    print("Middle")
} else {
    print("Low")
}